lib/origen_sim/simulator.rb

```diff
@@ -341,6 +359,8 @@ def run_dir
         d
       when :synopsys
         wave_dir
+      when :veridi
```

Comment:
See this typo, which is easy enough to fix, but I guess means that it is currently running frim the tmp_dir, is that OK or do you need it to be the wave_dir?